nondex-test/src/test/java/edu/illinois/nondex/core/AbstractCollectionTest.java

```diff
@@ -46,7 +48,7 @@ protected void assertParameterized(T ds, Object derived, String str) {
             case FULL:
                 String tempStr = derived.toString();
                 Assert.assertNotEquals("FULL is improperly running", str, tempStr);
-                this.assertEqualstUnordered("Does not match permutation", str, tempStr);
+                this.assertEqualsUnordered("Does not match permutation", str, tempStr);
```

Comment:
Any idea why they ever called it `Equalst`? Was it a misspelling or some joke about `EqualsString`? Does the Git log show anything useful?

Reply:
Hello Professor, I checked the git blame and from that I can infer that it was initially called as such. this was the first time when the function was commited/created
https://github.com/TestingResearchIllinois/NonDex/commit/d4f6e033687d0203a4f6557405345dc9422b240a#diff-b6d9573a5585dde09208a701b942a04b7e8017e4916381997baa33cf6d32edabR38